### PR TITLE
All entities should now use the visible fields method

### PR DESCRIPTION
**Pull request by atothhpe**
Hi, the old solution does not fix the dropdown for the exceptions. So they have more fields there than they should. @kdrtibor is that correct? I'm not 100% sure.

Anyway the problem is the subtype field, if we make sure it's set, i.e. do it manually. The visible fields method will work on everything.

WDYT?

### CODE REVIEW

**Comment by atothhpe** [comment]
@kdrtibor gonna keep the PR open, since the dropdown still has too many fields, maybe fix that on this branch as well

**Comment by kdrtibor** [comment]
I checked and compared the results on both branches, this one seems to have less fields, checked and the number of field matches the list in octane.

File: src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/GeneralEntityDetailsPanel.java [hunk 1]
```diff
@@ -291,7 +292,10 @@ public void activateCommentsCollapsible() {
     public void setFieldSelectButton(EntityDetailPresenter.SelectFieldsAction fieldSelectButton) {
         headerPanel.setFieldSelectButton(fieldSelectButton);
         fieldsPopup = new FieldsSelectFrame(defaultFields.get(Entity.getEntityType(entityModel)),
-                fields,
+                fields.stream().filter(e -> !"subtype".equals(e.getName())
+                                            &&!"description".equals(e.getName())
+                                            &&!"phase".equals(e.getName())
+                                            &&!"name".equals(e.getName())).collect(Collectors.toList()),
```
Comment (atothhpe) [inline src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/GeneralEntityDetailsPanel.java]: !Arrays.asList(1,2,3,4).contains(e.getName())

**Comment by kdrtibor** [comment]
Name Phase Description Subtype filtered + field panels have equal sizes

**Pull request merged**